docs/nodes/maintain/avalanchego-config-flags.md

```diff
@@ -514,61 +519,69 @@ Time between gossiping accepted frontiers. Defaults to `10s`.
 
 Timeout before killing an unresponsive chain. Defaults to `5s`.
 
-#### `--creation-tx-fee` (int):
+#### `--create-asset-tx-fee` (int):
+
+Transaction fee, in nAVAX, for transactions that create new assets. This can only be changed on a local network.
+
+#### `--create-subnet-tx-fee` (int):
+
+Transaction fee, in nAVAX, for transactions that create new subnets. This can only be changed on a local network.
+
+#### `--create-blockchain-tx-fee` (int):
 
-Transaction fee, in nAVAX, for transactions that create new state. Defaults to `1000000` nAVAX (.001 AVAX) per transaction.
+Transaction fee, in nAVAX, for transactions that create new blockchains. This can only be changed on a local network.
```

Comment:
```suggestion
Transaction fee, in nAVAX, for transactions that create new blockchains. Defaults to `1000000000` nAVAX (1 AVAX) per transaction. This can only be changed on a local network.
```